dot/peerset/peerstate.go

```diff
@@ -88,19 +88,11 @@ func newNode(n int) *node {
 	}
 }
 
-func (n *node) getReputation() Reputation {
-	return n.rep
```

Comment:
nit rename the `rep` field to `reputation`? Now that we use it in other files, it's a bit non-intuitive of what it is.